### PR TITLE
Release notes for 5.7.0

### PR DESCRIPTION
**Pull request by samjwu**
Note: All updates in this PR are pulled from the CHANGELOG of ROCm projects

eg: https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/develop/CHANGELOG.md

### CODE REVIEW

**Comment by samjwu** [comment]
> @samjwu The changelog has comprehensive updates for Libraries. Thanks for the updates. How about the other components? When can we expect to review them?

This PR includes updates for projects listed here: https://github.com/RadeonOpenCompute/ROCm/blob/develop/default.xml

Some are not included in these release notes because they either do not have a CHANGELOG.md, a 5.7 release branch, or both. We would have to ask the component owners for these updates.

**Comment by saadrahim** [comment]
Let's wait to merge this until other release milestones are completed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,295 @@ The release notes for the ROCm platform.
...
+| rocWMMA |  ⇒ [1.2.0](https://github.com/ROCmSoftwarePlatform/rocWMMA/releases/tag/rocm-5.7.0) |
+| Tensile |  ⇒ [4.38.0](https://github.com/ROCmSoftwarePlatform/Tensile/releases/tag/rocm-5.7.0) |
+
+#### hipBLAS 1.1.0
+
+hipBLAS 1.1.0 for ROCm 5.7.0
+
+##### Changed
+
+- updated documentation requirements
```
Comment (Rmalavally) [inline CHANGELOG.md:48]: What requirements? It would help users if we provided more details.

Comment (Rmalavally) [inline CHANGELOG.md:48]: Uppercase (Updated) for consistency

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,295 @@ The release notes for the ROCm platform.
...
+rocBLAS 3.1.0 for ROCm 5.7.0
+
+##### Added
+
+- yaml lock step argument scanning for rocblas-bench and rocblas-test clients. See Programmers Guide for details.
+- rocblas-gemm-tune is used to find the best performing GEMM kernel for each of a given set of GEMM problems.
+
+##### Fixed
+
+- make offset calculations for rocBLAS functions 64 bit safe.  Fixes for very large leading dimensions or increments potentially causing overflow:
```
Comment (Rmalavally) [inline CHANGELOG.md:145]: Uppercase (Make) for consistency

File: CHANGELOG.md (hunk 1 above)
Comment (samjwu) [reply CHANGELOG.md:48]: This is automatically pulled from https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/develop/CHANGELOG.md

**Review by saadrahim — APPROVED** [review]

**Pull request merged**